src/Validate.elm

```diff
@@ -3,14 +3,18 @@ module Validate exposing (..)
 
 import Dict
 import Errors
-import Formula
+import Formula exposing (Formula(..))
+import Formula.Signed exposing (Signed(..))
```

Comment:
I wonder it it wouldn't be nicer to `import Formula.Signed as Signed exposing (Signed(..))`. @yoyko?